backend/slack_commands/service.py

```
@@ -0,0 +1,45 @@
+        if handler is None:
+            return SlackResponse(
+                text=(
+                    f"Unknown command: `{command.subcommand}`. "
+                    f"Type `{command.command} help` for available commands."
+                ),
+            )
+
+        try:
+            return handler.handle(command)
+        except Exception as e:
+            logger.error(
```

Ruff (0.15.15)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)